modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-active-claims/index.css

```diff
@@ -0,0 +1,10 @@
+.active-claims-container .active-claims-title::after {
+	background-color: var(--primary);
```

Comment:
I believe that we have this color on Dialect.

Reply:
This color refers to the component created by after, so it is not possible to specify a color with the dialect for it in the className.